psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -3497,8 +3725,8 @@ void FISAPT::find() {
 
     // => ESPs <= //
 
-    dfh_->add_disk_tensor("WBar", std::make_tuple(nB + nb, na, nr));
-    dfh_->add_disk_tensor("WAbs", std::make_tuple(nA + na, nb, ns));
+    dfh_->add_disk_tensor("WBar", std::make_tuple(nB + nb + 1, na, nr)); // add entry for external potentials
```

Comment:
Same definition concern as earlier.

Reply:
This is the same idea as above. In addition to the nuclei and orbital contributions, we also have contributions from point charges.